app/src/main/kotlin/com/simplemobiletools/gallery/pro/helpers/Config.kt

```diff
@@ -363,6 +363,10 @@ class Config(context: Context) : BaseConfig(context) {
         get() = prefs.getString(LAST_FILEPICKER_PATH, "")!!
         set(lastFilepickerPath) = prefs.edit().putString(LAST_FILEPICKER_PATH, lastFilepickerPath).apply()
 
+    var lastCopyPath: String
```

Comment:
after accepting lets move this, the constant and getOtherFolderOpeningPath to commons, so that we dont have to copypaste it into the file manager